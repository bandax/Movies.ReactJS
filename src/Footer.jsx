import React, { PureComponent } from 'react';

export default class Footer extends PureComponent {
    render(){
        return <h4>Contact: administrator@movies.com</h4>
    }
}