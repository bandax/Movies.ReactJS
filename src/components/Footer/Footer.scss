.footer {
    background-color: #424242;
    text-align: center;
    
}
