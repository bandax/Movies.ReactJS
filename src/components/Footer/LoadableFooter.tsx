import * as React from 'react';
import Loadable from 'react-loadable';
import Loading from '../Loading/Loading';

const LoadableFooter = Loadable({
  loader: () => import('./Footer'),
  loading: Loading,
});

export default class FooterSplit extends React.Component {
  render() {
    return <LoadableFooter />;
  }
}
