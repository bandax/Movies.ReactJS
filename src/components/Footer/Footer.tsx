import * as React from 'react';
import './Footer.scss';

const Footer: React.FunctionComponent = () => (
  <div className="footer">
    <img src="../../../assets/netflix-roulette.PNG" />
  </div>
);

export { Footer };
