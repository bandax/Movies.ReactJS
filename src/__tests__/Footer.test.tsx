import * as React from 'react';
import { Footer } from '../components/Footer/Footer';
import { render, fireEvent, waitFor, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

describe('<Footer />', () => {
  it('render a Footer Component', async () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
