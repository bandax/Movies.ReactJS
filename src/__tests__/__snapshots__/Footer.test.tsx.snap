// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> render a Footer Component 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <img
      src="../../../assets/netflix-roulette.PNG"
    />
  </div>
</DocumentFragment>
`;
